src/api/routes/myGroups.ts

```diff
@@ -17,6 +17,13 @@ function isSubset<T>(superset: Set<T>, subset: Set<T>): boolean {
   return true;
 }
 
+function validMeetingLink(group: Group) {
+  // meeting is valid for 31 days after start time
+  // check if the link is created within 30 days
+  const checkPeriod = 2592000; // 30 days unix timestamp
```

Comment:
use js moment library to do the calculation: https://momentjs.com/docs/#/manipulating/

`if today > moment(updatedAt).add(30, 'days') then...`